cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -491,21 +491,6 @@ define([
             AjaxHelpers.expectNoRequests(requests);
         });
 
-        it('should have appropriate class names on focus/blur', function (done) {
```

Comment:
Yes, they noted specific problems with Firefox, which is the browser we are using in Jenkins. I don't personally think it is worth trying to make those tests conditionally run in other browsers (and I am skeptical that they will pass reliably on other browsers). And the reality is that nobody ever comes back to fix unit tests that are disabled with "xit", so it just adds cruft. But I'll let @andy-armstrong and @bjacobel settle on the ultimate decision.